syntax = "proto2";
package protobuf;
message FeatureRequest {
    required int64 user_id = 1;
}